CODE_OF_CONDUCT.md

```diff
@@ -23,23 +14,29 @@ All recipients of reports commit to:
     * **Node-RED:** <team@nodered.org>
     * **Webdriver.io:** [contact TSC reps](https://github.com/webdriverio/webdriverio/blob/HEAD/AUTHORS.md)
     * **Webhint:** <support@webhint.io>
-2. For every other OpenJS Foundation project, please email <report@lists.openjsf.org>. The Cross Project Council (CPC) is responsible for managing these reports.
+    
+    In case of a conflict of interest or if the report is not handled appropriately, you may [escalate the report][escalation] to the [Code of Conduct Team (CoC Team)][CoC Team] by emailing <coc-escalation@lists.openjsf.org> or by contacting any of its members.
+2.  **To report any other incident** (for example: in an OpenJS Foundation project not listed above, in a Collab Space, in the Cross Project Council, or during an event), please email the [Code of Conduct Team (CoC Team)][CoC Team] at <report@lists.openjsf.org> or contact any of its members.
+
+### Appeal a decision
 
+The victim(s) and offender(s) have thirty (30) days to appeal the decision made in response to an incident report if they believe the process was not properly followed. Appeals are not suspensive.
```

Comment:
shouldn't this be the reporter rather than "victim"?

and is a "victim" a necessary component of a CoC violation?  "victim" is a loaded word, and may not apply to all CoC violations

Reply:
Fixed the language issue.

> shouldn't this be the reporter rather than "victim"?

Unless the reporter is themself harmed, I don't think they should have the right to appeal. They're not a party in the process, imho.